src/views/home/UserInput.js

```diff
@@ -12,14 +12,22 @@ const inputStyle = {
 
 export default class UserInput extends Component {
   state = {
-    name: 'Example ticket',
     url: '',
     screenshot: {
       height: '',
```

Comment:
We can also set userAgent as part of the request, iirc the Target folks were talking about it at the presentation.

Reply:
Sounds good! I think we're going to have rewrite that create ticket request.